src/Makefile.am

```diff
@@ -778,8 +782,42 @@ bitcoin_util_LDADD = \
 #
 
 # bitcoin-chainstate binary #
-bitcoin_chainstate_SOURCES = \
-  bitcoin-chainstate.cpp \
+bitcoin_chainstate_SOURCES = bitcoin-chainstate.cpp
+bitcoin_chainstate_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
+bitcoin_chainstate_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+bitcoin_chainstate_LDFLAGS = $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS) $(PTHREAD_FLAGS)
+bitcoin_chainstate_LDADD = $(LIBBITCOINKERNEL)
+#
+
+# bitcoinconsensus library #
```

Comment:
In commit "build: Extract the libbitcoinkernel library" (764c1f18c031166b406be439486956fb84d3b9fe)

Not very important, but if targets were declared in order: bitcoin_chainstate, bitcoinkernel, bitcoinconsensus, less code would be moving around and related chainstate/kernel declarations would be closer together

Reply:
Fixed as of 3961a789735c874ed9b3ff4bbd64c2b850282c07